staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -52,7 +52,7 @@ type fakeObjectDefaulter struct{}
 
 func (d *fakeObjectDefaulter) Default(in runtime.Object) {}
 
-func NewTestFieldManager() *fieldmanager.FieldManager {
+func NewTestFieldManager() fieldmanager.FieldManager {
```

Comment:
That's conflicting with what I'm doing :cry:

Reply:
I can rebase on top of yours